pkg/kubelet/images/image_manager_test.go

```diff
@@ -631,14 +631,18 @@ func TestShouldPullImage(t *testing.T) {
 		},
 	}
 
-	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.KubeletEnsureSecretPulledImages, true)()
-
-	for i, test := range tests {
-		sp := shouldPullImage(test.container, test.imagePresent, test.pulledBySecret, test.ensuredBySecret)
-		if utilfeature.DefaultFeatureGate.Enabled(features.KubeletEnsureSecretPulledImages) {
-			assert.Equal(t, test.expectedWithFGOn, sp, "TestCase[%d]: %s ensured image fg enabled", i, test.description)
-		} else {
+	t.Run("disabled_features", func(t *testing.T) {
```

Comment:
nit: is there a reason not to mention which feature? I only see one being disabled/enabled

Reply:
Add the feature gate name here.